src/main/java/com/incognia/onboarding/SignupAssessment.java

```diff
@@ -10,6 +12,7 @@ public class SignupAssessment {
   UUID id;
   UUID requestId;
   Assessment riskAssessment;
+  List<Reason> reasons;
```

Comment:
This can be specified as a `Set` (it is a `Set` in the response DTO).

Reply:
sure